README.md

```diff
@@ -166,19 +174,37 @@ import { build } from 'esbuild'
 
 build({
   plugins: [
-    require('./unplugin-feature').esbuild({ /* options */ }),
+    require('./unplugin-feature').esbuild({
+      /* options */
+    }),
```

Comment:
Could you revert the formatting changes here? Thanks.

Reply:
Formatting problem has been fixed